test/source/tests/tests/compose.ts

```diff
@@ -409,6 +409,19 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
       }
     }));
 
+    ava.default('compose[global:compose] - standalone - load contacts through API', testWithNewBrowser(async (t, browser) => {
+      await BrowserRecipe.setUpCommonAcct(t, browser, 'compose');
+      const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compose');
+      await composePage.click('@action-expand-cc-bcc-fields');
+      await composePage.type('@input-to', 'contact');
+      let contacts = await composePage.waitAny('@container-contacts');
+      expect(await PageRecipe.getElementPropertyJson((await contacts.$('ul li:first-child'))!, 'textContent')).to.eq('No Contacts Found');
+      const oauthPopup = await browser.newPageTriggeredBy(t, () => composePage.click('@action-auth-with-contacts-scope'), 'test.ci.compose@org.flowcrypt.com');
+      await OauthPageRecipe.google(t, oauthPopup, 'test.ci.compose@org.flowcrypt.com', 'approve');
+      contacts = await composePage.waitAny('@container-contacts');
+      expect(await PageRecipe.getElementPropertyJson((await contacts.$('ul li:first-child'))!, 'textContent')).to.eq('contact.test@flowcrypt.com');
```

Comment:
Here, it would be good to also test that after we reload the compose page, it remembers the newly allowed scope, and will right away just load the contacts.